<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Qiancheng Xiong</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="description" content="Profile page of Qiancheng Xiong, researcher at The Wyss Institute for Biologically Inspired Engineering at Harvard University.">
        <meta name="robots" content="index, follow" />
        <meta name="googlebot" content="index, follow, max-snippet:-1, max-image-preview:large, max-video-preview:-1" />
        <meta name="bingbot" content="index, follow, max-snippet:-1, max-image-preview:large, max-video-preview:-1" />

        <meta name="msvalidate.01" content="0477ED63E0C6D968BB8B595E3C610279" />
        <meta http-equiv="Content-Security-Policy" content="
          default-src 'self';
          script-src 'self' 'unsafe-inline' https://www.googletagmanager.com https://code.jquery.com https://cdnjs.cloudflare.com https://stackpath.bootstrapcdn.com;
          style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://stackpath.bootstrapcdn.com https://cdnjs.cloudflare.com;
          font-src 'self' https://fonts.gstatic.com https://cdnjs.cloudflare.com;
          img-src 'self' data:;
          connect-src 'self';
          object-src 'none';
        ">




        <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
        <link rel="manifest" href="/site.webmanifest">
        <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
        <meta name="msapplication-TileColor" content="#da532c">
        <meta name="theme-color" content="#ffffff">


        <script type="application/ld+json">
        {
          "@context": "https://schema.org/",
          "@type": "Person",
          "name": "Qiancheng Xiong",
          "url": "https://www.qcxiong.com/",
          "image": "https://www.qcxiong.com/img/qiancheng-xiong.jpg",
          "sameAs": [
            "https://www.linkedin.com/in/qcxiong/",
            "https://scholar.google.com/citations?user=qRvTgzgAAAAJ",
            "https://github.com/xiongqc",
            "https://orcid.org/0000-0002-9897-7391"
          ],
          "worksFor": {
            "@type": "CollegeOrUniversity",
            "name": "The Wyss Institute for Biologically Inspired Engineering at Harvard University"
          }  
        }
        </script>

        <!--CSS styles-->
        <link rel="stylesheet" href="css/bootstrap.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">

        <!-- Perfect Scrollbar CSS from CDN -->
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/1.5.5/css/perfect-scrollbar.min.css">


        <!-- Magnific Popup CSS from CDN -->
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/1.1.0/magnific-popup.min.css">

        
        <!-- Custom CSS Files -->
        <link rel="stylesheet" href="css/style.css">
        <link id="theme-style" rel="stylesheet" href="css/styles/blue.css">
        <!--/CSS styles-->
	    
        <!--Javascript files-->

        <!-- jQuery (Latest version from CDN) -->
        <script src="https://code.jquery.com/jquery-3.6.0.min.js" defer></script>

        <!-- Bootstrap JS (Latest version from CDN) -->
        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.bundle.min.js" defer></script>

        <!-- GSAP (TweenMax and ScrollToPlugin) from CDN -->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.5/gsap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.5/ScrollToPlugin.min.js" defer></script>

        <!-- jQuery Plugins from CDN -->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.touchswipe/1.6.19/jquery.touchSwipe.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/1.1.0/jquery.magnific-popup.min.js" defer> 
        </script>

        <!-- Modernizr (Custom build; keep local if necessary) -->
        <script src="js/modernizr.custom.63321.js" defer></script>
        <script src="js/jquery.stellar.min.js" defer></script>
        <script src="js/jquery.mixitup.min.js" defer></script>
        <script src="js/perfect-scrollbar-0.4.5.with-mousewheel.min.js" defer></script>

        <!-- Plugins not available via CDN (Keep local) -->
        <script src="js/jquery.carouFredSel-6.2.1-packed.js" defer></script>
        <script src="js/jquery.dropdownit.js" defer></script>

        <!-- Custom scripts -->
        <script src="js/custom.js" defer></script>

        <!--/Javascript files-->


<!-- Open Graph Tags -->
<meta property="og:title" content="Qiancheng Xiong">
<meta property="og:description" content="Profile page of Qiancheng Xiong, researcher at The Wyss Institute.">
<meta property="og:image" content="https://www.qcxiong.com/img/qiancheng-xiong.jpg">
<meta property="og:url" content="https://www.qcxiong.com/">
<meta property="og:type" content="profile">

<!-- Twitter Card Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:title" content="Qiancheng Xiong">
<meta name="twitter:description" content="Profile page of Qiancheng Xiong, researcher at The Wyss Institute.">
<meta name="twitter:image" content="https://www.qcxiong.com/img/qiancheng-xiong.jpg">


<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Q39F2LW18V"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Q39F2LW18V');
</script>

	    
</head>

    <body>

        <div id="wrapper">
            <a href="#sidebar" class="mobilemenu"><i class="icon-reorder"></i></a>

            <div id="sidebar">
                <div id="main-nav">
                    <div id="nav-container">
                        <div id="profile" class="clearfix">
                            <div class="portrate hidden-xs"></div>
                            <div class="title">
                                <h2>Qiancheng Xiong</h2>
                                <h3>Wyss Institute</h3> 
                                <h3>Harvard University</h3>
                            </div>
                            
                        </div>
                        <ul id="navigation">
                            <li>
                              <a href="#biography">
                                <div class="icon icon-user"></div>
                                <div class="text">About Me</div>
                              </a>
                            </li>  
                            <!--
                            <li>
                              <a href="#research">
                                <div class="icon icon-book"></div>
                                <div class="text">Research</div>
                              </a>
                            </li>
                            -->
                            <li>
                              <a href="#publications">
                                <div class="icon icon-edit"></div>
                                <div class="text">Publications</div>
                              </a>
                            </li>  
                            <li>
                              <a href="#contact">
                                <div class="icon icon-calendar"></div>
                                <div class="text">Contact</div>
                              </a>
                            </li>
                            <!--
                            <li class="external">
                              <a href="xiongqc.github.io/cv/CV_Qiancheng_Xiong caa 050118.pdf" download>
                                  <div class="icon icon-download-alt"></div>
                                  <div class="text">Curricula Vitae</div>
                              </a>
                            </li>
                            -->
                        </ul>
                    </div>        
                </div>
                
                <div class="social-icons">
                    <ul>
                        <!--
                        <li><a href="https://www.facebook.com/qiancheng"><i class="icon-facebook"></i></a></li>
                        -->
                        <li><a href="https://twitter.com/qcxiong"><i class="icon-twitter"></i></a></li>
                        <li><a href="https://www.linkedin.com/in/qcxiong"><i class="icon-linkedin"></i></a></li>
                    </ul>
                </div>    
            </div>

            <div id="main">
            
                <div id="biography" class="page home" data-pos="home">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                
                                <div class="row">
                                    <div class="col-sm-2 visible-sm"></div>
                                    <div class="col-sm-8 col-md-5">
                                        <div class="biothumb">
                                            <img alt="image" src="img/personal/bio.jpg"  class="img-responsive">
                                        </div>
                                        
                                    </div>
                                    <div class="clearfix visible-sm visible-xs"></div>
                                    
                                    <div class="col-sm-12 col-md-7">
                                        <h3 class="title">Biography</h3>
                                        <p>I am a Research Fellow in the William Shih Lab at the Wyss Institute and Dana-Farber Cancer Institute, where my research primarily focuses on advancing RNA delivery methodologies through the innovative use of nucleic acid origami nanotechnology. I hold a PhD in Cell Biology from Yale University and a BA in Natural Sciences from Cambridge University.</p>
                                    </div>  
                                    
                                </div>
                            </div>        
                        </div>
                    </div>

                    <div class="pagecontents">
                        <div class="section color-1">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-5 col-md-offset-1">
                                        <div class="title text-center">
                                            <h3>Work Experience</h3>
                                        </div>
                                        <ul class="ul-dates">
                                        
                                            <li>
                                                <div class="dates">
                                                    <span>Present</span>
                                                    <span>2023</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Research Fellow</h4>
                                                    <p><em>Harvard University</em>, The Wyss Institute for Biologically Inspired Engineering</p>
                                                    <p><em>Harvard Medical School</em>, Dana-Farber Cancer Institute</p>
                                                </div>
                                            </li>


                                            <li>
                                                <div class="dates">
                                                    <span>2017</span>
                                                    <span>2016</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Research Officer</h4>
                                                    <p><em>Agency for Science, Technology and Research</em>, Institute of Molecular and Cell Biology</p>
                                                </div>
                                            </li>

                                        
                                            <li>
                                                <div class="dates">
                                                    <span>2012</span>
                                                    <span>2011</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Detachment Commander</h4>
                                                    <p><em>Singapore Armed Forces</em>, Singapore Armoured Regiment</p>
                                                    <a href="/testimonial/Testimonial_SAF.pdf" target="_blank">
                                                    <p>
                                                    <strong><i class="icon-thumbs-up"></i></strong>
                                                	<em>View Testimonial</em>
                                                	</p>
                                                	</a>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dates">
                                                    <span>2010</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Intern</h4>
                                                    <p><em>Singapore Ministry of Defence</em>, Defence Psychology Department</p>
													<a href="/testimonial/Testimonial_MINDEF.pdf" target="_blank">
                                                    <p>
                                                    <strong><i class="icon-thumbs-up"></i></strong>
                                                	<em>View Testimonial</em>
                                                	</p>
                                                	</a>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                    <div class="col-md-5">
                                        <div class="title text-center">
                                            <h3>Education</h3>
                                        </div>
                                        <ul class="ul-card">
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">Ph.D.</span><span class="year">2023</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Ph.D. in Cell Biology, 2023</p>
                                                    <p class="waht">Master of Philosophy in Cell Biology, 2020</p>
                                                    <p class="where">Yale University</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">B.A.</span><span class="year">2016</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Bachelor of Arts in Natural Sciences</p>
                                                    <p class="where">University of Cambridge</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">Diploma</span><span class="year">2010</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Honours in Mathematics, Biology and Chemistry, and Major in Physics</p>
                                                    <p class="where">National University of Singapore</p> 
                                                    <p class="where">High School of Mathematics and Science</p>	
                                                </div>
                                            </li>
						                    <li>
                                                <div class="dy">
                                                    <span class="degree">Languages</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Chinese</p>
                                                    <p class="where">GCE O-Level Higher Chinese</p> 
                                                </div>
							
						                        <div class="description">
                                                    <p class="waht">Japanese</p>
                                                    <p class="where">GCE O-Level Japanese</p> 
                                                </div>
						                        <div class="description">
                                                    <p class="waht">French</p>
                                                    <p class="where">CEFRL A2 Level</p> 
                                                </div>
                                            </li>
                                        </ul>
                                    </div>    
                                </div>    
                            </div>
                                
                        </div>

                        <div class="section color-2">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-10 col-md-offset-1">
                                        <div class="title text-center">
                                            <h3>Honors, Awards and Grants</h3>
                                        </div>
                                        <ul class="timeline">
                                            <li class="open">
                                                <div class="date">2023</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">Foresight Institute Feynman Prizes - Distinguished Student Award</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/foresight.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            Recipient of Foresight Institute's Distinguished Student Award for notable contributions to nanotechnology development and understanding.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>
                                            <li class="open">
                                                <div class="date">2023</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">Northpond Director’s Fund</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/wyss.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            Recipient of a grant supporting early-stage research at the Wyss Institute.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>
                                            <li class="open">
                                                <div class="date">2023</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">A*STAR International Fellowship</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/ASTAR.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            Awarded for post-doctoral training at top overseas institution to deepen research expertise and expand global network.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li> 
                                            <li class="open">
                                                <div class="date">2017</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">A*STAR Graduate Scholarship</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/ASTAR.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            The scholarship fully funds up to four years of PhD studies. 
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>        
                                            <li class>
                                                <div class="date">2011</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">National Science Scholarship (BS)</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/ASTAR.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            An undergraduate scholarship that grooms talents for a PhD education.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>                            
                    </div>
                </div>  
                <!--
                <div id="research" class="page">
                    <div class="pageheader">

                        <div class="headercontent">

                            <div class="section-container">
                                <h2 class="title">Research Summary</h2>
                            
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>Prof Zeng Qi identified PRL-3 in 1998 and her group later demonstrated that the phosphatase plays a causal role in various types of cancer. Together with Abdul from the group, Qiancheng is working to further elucidate the molecular details of how PRL-3 promotes cancer. </p>
                                    </div>
                                    <div class="col-md-4">
                                        <div class="subtitle text-center">
                                            <h3>Interests</h3>
                                        </div>
                                        <ul class="ul-boxed list-unstyled">
                                            <li>Genetics</li>
                                            <li>Bioinformatics</li>
                                            <li>Cancer</li>
                                            <li>Infectious Diseases</li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    
                    <div class="pagecontents">
                        <div class="section color-2">
                            <div class="section-container">
                                <div class="title text-center">
                                    <h3>Research Experience</h3>
                                </div>
                                <div class="row">
                                    <div class="col-md-12">
                                        <ul class="ul-withdetails">
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p2.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Molecular Evolution of Rotaviruses</h3>
                                                            <p><em>January - March 2016</em></p>
                                                            <p>Department of Veterinary Medicine, University of Cambridge</p>
                                                            <p>Lab of Simon Frost</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Rotavirus is a major cause of acute gastroenteritis and child mortality worldwide. Through estimating selection pressures and mutation rates at both the gene level and the site-by-site level, I demonstrated evolutionary differences between rotavirus species, proteins and hosts. In particular, higher mutation rate and diversifying selection estimates for species B may facilitate its expansion to geographical regions outside Asia. Capsid protein VP6 was also found to be a possible vaccine target due to its low mutation rate and diversifying selection estimates. Separately, reconstructed effective population size and estimated mutation rates for species A showed no significant change in genetic variation after the introduction of modern vaccines in 2005, but new sites under diversifying selection were identified on its gene for the vaccine targeted protein VP4. Finally, reconstructed phylogenies showed ongoing zoonotic transmission in species A but not in species B and C.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p3.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Transcriptional Regulation by RNA Antiterminators</h3>
                                                            <p><em>July-August 2015</em></p>
                                                            <p>Institut de Génétique et de Biologie Moléculaire et Cellulaire</p>
                                                            <p>Lab of Albert Weixlbaumer</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Transcripts of putRNAs from phage HK022 can fold into two stem-loops to inhibit RNA polymerase backtracking and facilitate antitermination. To assist the lab in understanding the molecular and structural details of this process, I cloned mutant put in vivo reporter constructs with sGFP and lacZα, and compared the effect of antitermination by wild type putRNA and their mutants using in vivo reporter assays. I also produced putRNA using in vitro transcription and purified it for the lab’s later studies.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p4.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Prevalence of Wolbachia in Moths & Phylogenetic Comparison of Wolbachia and Moths</h3>
                                                            <p><em>September-October 2014</em></p>
                                                            <p>Department of Zoology, University of Cambridge</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Wolbachia infection provides insect hosts with virus resistance but at the cost of hosts’ fitness. To better understand the side effects of Wolbachia pest control measures, the relationship of Wolbachia with moth hosts was explored by my team using computational prevalence and phylogenetic analysis. This was performed with DNA extracted from moths caught at Madingley, as well as with sequence data from GenBank.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p5.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Nuclear Translocation of NFκB in Response to Viral Protein Expression</h3>
                                                            <p><em>June-August 2014</em></p>
                                                            <p>Singapore Immunology Network, Agency for Science, Technology and Research</p>
                                                            <p>Lab of Ren Ee Chee</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>In a co-localization study, I assisted the lab group by quantifying the relative amounts of NFκB nuclear translocation in cells that were over-expressing different variants of a viral protein using western blots. In addition, I generated plasmid constructs to assist two separate projects: to investigate how residues in a viral protein affect its function; and to create CRISPR knock-out cell lines for studying viral replication.</p>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>    
                    </div>
                </div>
                -->
                <div id="publications" class="page">
                    <div class="page-container">
                        <div class="pagecontents">
                            <div class="headercontent">
                                <div class="section-container">
                                    <h2 class="title">Publications</h2>
                                </div>
                                <div class="section color-1" id="filters">
                                    <div class="section-container">
                                        <div class="row">
                                            
                                            <div class="col-md-3">
                                                <h3>Filter by type:</h3>
                                            </div>
                                            <div class="col-md-6">
                                                <select id="cd-dropdown" name="cd-dropdown" class="cd-select">
                                                    <option class="filter" value="all" selected>All types</option>
                                                    <option class="filter" value="jpaper">Original Research</option>
                                                    <option class="filter" value="cpaper">Commentary</option>
                                                    <!-- <option class="filter" value="bookchapter">Book Chapter</option> -->
                                                    <!-- <option class="filter" value="book">Book</option> -->
                                                    <option class="filter"value="report">Review</option>
                                                    <option class="filter" value="tpaper">Patent</option>
                                                </select>
                                            </div>
                                            
                                            <div class="col-md-3" id="sort">
                                                <span>Sort by year:</span>
                                                <div class="btn-group pull-right"> 

                                                    <button type="button" data-sort="data-year" data-order="desc" class="sort btn btn-default"><i class="icon-sort-by-order"></i></button>
                                                    <button type="button" data-sort="data-year" data-order="asc" class="sort btn btn-default"><i class="icon-sort-by-order-alt"></i></button>
                                                </div>
                                            </div>    
                                        </div>
                                    </div>
                                </div>
                            </div>
                            <div class="section color-2" id="pub-grid">
                                <div class="section-container">
                                    <div class="row">
                                        <div class="col-md-12">
                                            <div class="pitems">
                                            <!-- Add new items after this line -->
                                            <!-- paper separator -->
                                                <div class="item mix jpaper" data-year="2024">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.7150/thno.93127" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">PRL1 and PRL3 promote macropinocytosis via its lipid phosphatase activity</h4>
                                                        <div class="pubauthor"><strong>Zu Ye</strong>, Chee Ping Ng, Haidong Liu, Qimei Bao, Shengfeng Xu, Dan Zu, Yanhua He, Yixing Huang, Abdul Qader Omer Al-Aidaroos, Ke Guo, Jie Li, Lai Ping Yaw, Qiancheng Xiong, Min Thura, Weihui Zheng, Fenghui Guan, Xiangdong Cheng, Yin Shi, Qi Zeng</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Theranostics, Volume 14, Issue 9, 27 May 2024, Pages 3976–4050</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>PRL1 and PRL3, members of the protein tyrosine phosphatase family, have been associated with cancer metastasis and poor prognosis. Despite extensive research on their protein phosphatase activity, their potential role as lipid phosphatases remains elusive. We conducted comprehensive investigations to elucidate the lipid phosphatase activity of PRL1 and PRL3 using a combination of cellular assays, biochemical analyses, and protein interactome profiling. Functional studies were performed to delineate the impact of PRL1/3 on macropinocytosis and its implications in cancer biology. Our study has identified PRL1 and PRL3 as lipid phosphatases that interact with phosphoinositide (PIP) lipids, converting PI(3,4)P2 and PI(3,5)P2 into PI(3)P on the cellular membranes. These enzymatic activities of PRLs promote the formation of membrane ruffles, membrane blebbing and subsequent macropinocytosis, facilitating nutrient extraction, cell migration, and invasion, thereby contributing to tumor development. These enzymatic activities of PRLs promote the formation of membrane ruffles, membrane blebbing and subsequent macropinocytosis. Additionally, we found a correlation between PRL1/3 expression and glioma development, suggesting their involvement in glioma progression. Combining with the knowledge that PRLs have been identified to be involved in mTOR, EGFR and autophagy, here we concluded the physiological role of PRL1/3 in orchestrating the nutrient sensing, absorbing and recycling via regulating macropinocytosis through its lipid phosphatase activity. This mechanism could be exploited by tumor cells facing a nutrient-depleted microenvironment, highlighting the potential therapeutic significance of targeting PRL1/3-mediated macropinocytosis in cancer treatment.</p>
                                                    </div>
                                                </div>
						                    <!-- paper separator -->
						                        <div class="item mix report" data-year="2023">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1021/acs.chemrev.3c00028" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Recent Advances in DNA Origami-Engineered Nanomaterials and Applications</h4>
                                                        <div class="pubauthor"><strong>Pengfei Zhan</strong>, Andreas Peil, Qiao Jiang, Dongfang Wang, Shikufa Mousavi, Qiancheng Xiong, Qi Shen, Yingxu Shang, Baoquan Ding, Chenxiang Lin, Yonggang Ke, Na Liu</div>
                                                        <div class="pubcite"><span class="label label-success">Review</span>Chemical Reviews, Volume 123, Issue 7, 29 March 2023, Pages 3976–4050</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>DNA nanotechnology is a unique field, where physics, chemistry, biology, mathematics, engineering, and materials science can elegantly converge. Since the original proposal of Nadrian Seeman, significant advances have been achieved in the past four decades. During this glory time, the DNA origami technique developed by Paul Rothemund further pushed the field forward with a vigorous momentum, fostering a plethora of concepts, models, methodologies, and applications that were not thought of before. This review focuses on the recent progress in DNA origami-engineered nanomaterials in the past five years, outlining the exciting achievements as well as the unexplored research avenues. We believe that the spirit and assets that Seeman left for scientists will continue to bring interdisciplinary innovations and useful applications to this field in the next decade.</p>
                                                    </div>
                                                </div>
                                            <!-- paper separator -->
                                                <div class="item mix jpaper" data-year="2023">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1073/pnas.2202815120" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">The capsid lattice engages a bipartite NUP153 motif to mediate nuclear entry of HIV-1 cores</h4>
                                                        <div class="pubauthor"><strong>Qi Shen</strong>, Sushila Kumari, Chaoyi Xu, Sooin Jang, Jiong Shi, Ryan C. Burdick, Lev Levintov, Qiancheng Xiong, Chunxiang Wu, Swapnil C. Devarkar, Taoran Tian, Therese N. Tripler, Yingxia Hu, Shuai Yuan, Joshua Temple, Qingzhou Feng, C. Patrick Lusk, Christopher Aiken, Alan N. Engelman, Juan R. Perilla, Vinay K. Pathak, Chenxiang Lin, Yong Xiong</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Proceedings of the National Academy of Sciences, Volume 120, Issue 13, 21 March 2023, Article e2202815120</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Increasing evidence has suggested that the HIV-1 capsid enters the nucleus in a largely assembled, intact form. However, not much is known about how the cone-shaped capsid interacts with the nucleoporins (NUPs) in the nuclear pore for crossing the nuclear pore complex. Here, we elucidate how NUP153 binds HIV-1 capsid by engaging the assembled capsid protein (CA) lattice. A bipartite motif containing both canonical and noncanonical interaction modules was identified at the C-terminal tail region of NUP153. The canonical cargo-targeting phenylalanine-glycine (FG) motif engaged the CA hexamer. By contrast, a previously unidentified triple-arginine (RRR) motif in NUP153 targeted HIV-1 capsid at the CA tri-hexamer interface in the capsid. HIV-1 infection studies indicated that both FG- and RRR-motifs were important for the nuclear import of HIV-1 cores. Moreover, the presence of NUP153 stabilized tubular CA assemblies in vitro. Our results provide molecular-level mechanistic evidence that NUP153 contributes to the entry of the intact capsid into the nucleus.</p>
                                                    </div>
                                            </div>
                                            <!-- paper separator -->
                                            <div class="item mix jpaper" data-year="2023">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41594-023-00925-9" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Modeling HIV-1 nuclear entry with nucleoporin-gated DNA-origami channels</h4>
                                                        <div class="pubauthor"><strong>Qi Shen</strong>, Qingzhou Feng, Chunxiang Wu, Qiancheng Xiong, Taoran Tian, Shuai Yuan, Jiong Shi, Gregory J. Bedwell, Ran Yang, Christopher Aiken, Alan N. Engelman, C. Patrick Lusk, Chenxiang Lin, Yong Xiong</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Structural & Molecular Biology, 20 February 2023</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Delivering the virus genome into the host nucleus through the nuclear pore complex (NPC) is pivotal in human immunodeficiency virus 1 (HIV-1) infection. The mechanism of this process remains mysterious owing to the NPC complexity and the labyrinth of molecular interactions involved. Here we built a suite of NPC mimics—DNA-origami-corralled nucleoporins with programmable arrangements—to model HIV-1 nuclear entry. Using this system, we determined that multiple cytoplasm-facing Nup358 molecules provide avid binding for capsid docking to the NPC. The nucleoplasm-facing Nup153 preferentially attaches to high-curvature regions of the capsid, positioning it for tip-leading NPC insertion. Differential capsid binding strengths of Nup358 and Nup153 constitute an affinity gradient that drives capsid penetration. Nup62 in the NPC central channel forms a barrier that viruses must overcome during nuclear import. Our study thus provides a wealth of mechanistic insight and a transformative toolset for elucidating how viruses like HIV-1 enter the nucleus.</p>
                                                    </div>
                                            </div>
                                            <!-- paper separator -->
						<div class="item mix jpaper" data-year="2023">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1021/jacs.2c11226" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Functionalized DNA-Origami-Protein Nanopores Generate Large Transmembrane Channels with Programmable Size-Selectivity</h4>
                                                        <div class="pubauthor"><strong>Qi Shen</strong>, <strong>Qiancheng Xiong</strong>, Kaifeng Zhou, Qingzhou Feng, Longfei Liu, Taoran Tian, Chunxiang Wu, Yong Xiong, Thomas J. Melia, C. Patrick Lusk, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Journal of the American Chemical Society, Volume 145, Issue 2, 18 January 2023, Pages 1292–1300</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The DNA-origami technique has enabled the engineering of transmembrane nanopores with programmable size and functionality, showing promise in building biosensors and synthetic cells. However, it remains challenging to build large (>10 nm), functionalizable nanopores that spontaneously perforate lipid membranes. Here, we take advantage of pneumolysin (PLY), a bacterial toxin that potently forms wide ring-like channels on cell membranes, to construct hybrid DNA–protein nanopores. This PLY-DNA-origami complex, in which a DNA-origami ring corrals up to 48 copies of PLY, targets the cholesterol-rich membranes of liposomes and red blood cells, readily forming uniformly sized pores with an average inner diameter of ∼22 nm. Such hybrid nanopores facilitate the exchange of macromolecules between perforated liposomes and their environment, with the exchange rate negatively correlating with the macromolecule size (diameters of gyration: 8–22 nm). Additionally, the DNA ring can be decorated with intrinsically disordered nucleoporins to further restrict the diffusion of traversing molecules, highlighting the programmability of the hybrid nanopores. PLY-DNA pores provide an enabling biophysical tool for studying the cross-membrane translocation of ultralarge molecules and open new opportunities for analytical chemistry, synthetic biology, and nanomedicine.</p>
                                                    </div>
                                                </div>
						<!-- paper separator -->
                                                <div class="item mix jpaper" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1126/sciadv.add1830" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Actuating tension-loaded DNA clamps drives membrane tubulation</h4>
                                                        <div class="pubauthor"><strong>Longfei Liu</strong>, Qiancheng Xiong, Chun Xie, Frederic Pincet, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Science Advances, Volume 8, Issue 41, 12 October 2022, eadd1830</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Membrane dynamics in living organisms can arise from proteins adhering to, assembling on, and exerting force on cell membranes. Programmable synthetic materials, such as self-assembled DNA nanostructures, offer the capability to drive membrane-remodeling events that resemble protein-mediated dynamics but with user-defined outcomes. An illustrative example is the tubular deformation of liposomes by DNA nanostructures with purposely designed shapes, surface modifications, and self-assembling properties. However, stimulus-responsive membrane tubulation mediated by DNA reconfiguration remains challenging. Here, we present the triggered formation of membrane tubes in response to specific DNA signals that actuate membrane-bound DNA clamps from an open state to various predefined closed states, releasing prestored energy to activate membrane deformation. We show that the timing and efficiency of vesicle tubulation, as well as the membrane tube widths, are modulated by the conformational change of DNA clamps, marking a solid step toward spatiotemporal control of membrane dynamics in an artificial system.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41467-022-30878-4" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Omicron-specific mRNA vaccination alone and as a heterologous booster against SARS-CoV-2</h4>
                                                        <div class="pubauthor"><strong>Zhenhao Fang</strong>, <strong>Lei Peng</strong>, Renata Filler, Kazushi Suzuki, Andrew McNamara, Qianqian Lin, Paul A. Renauer, Luojia Yang, Bridget Menasche, Angie Sanchez, Ping Ren, Qiancheng Xiong, Madison Strine, Paul Clark, Chenxiang Lin, Albert I. Ko, Nathan D. Grubaugh, Craig B. Wilen, Sidi Chen</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Communications, Volume 13, Issue 1, 06 June 2022, Page 3250</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The Omicron variant of SARS-CoV-2 recently swept the globe and showed high level of immune evasion. Here, we generate an Omicron-specific lipid nanoparticle (LNP) mRNA vaccine candidate, and test its activity in animals, both alone and as a heterologous booster to WT mRNA vaccine. Our Omicron-specific LNP-mRNA vaccine elicits strong antibody response in vaccination-naïve mice. Mice that received two-dose WT LNP-mRNA show a > 40-fold reduction in neutralization potency against Omicron than WT two weeks post boost, which further reduce to background level after 3 months. The WT or Omicron LNP-mRNA booster increases the waning antibody response of WT LNP-mRNA vaccinated mice against Omicron by 40 fold at two weeks post injection. Interestingly, the heterologous Omicron booster elicits neutralizing titers 10-20 fold higher than the homologous WT booster against Omicron variant, with comparable titers against Delta variant. All three types of vaccination, including Omicron alone, WT booster and Omicron booster, elicit broad binding antibody responses against SARS-CoV-2 WA-1, Beta, Delta variants and SARS-CoV. These data provide direct assessments of an Omicron-specific mRNA vaccination in vivo, both alone and as a heterologous booster to WT mRNA vaccine.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1016/j.xcrm.2022.100634" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Variant-specific vaccination induces systems immune responses and potent <em>in vivo</em> protection against SARS-CoV-2</h4>
                                                        <div class="pubauthor"><strong>Lei Peng</strong>, <strong>Paul A Renauer</strong>, <strong>Arya Ökten</strong>, <strong>Zhenhao Fang</strong>, <strong>Jonathan J Park</strong>, <strong>Xiaoyu Zhou</strong>, Qianqian Lin, Matthew B Dong, Renata Filler, Qiancheng Xiong, Paul Clark, Chenxiang Lin, Craig B Wilen, Sidi Chen</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Cell Reports Medicine, Volume 3, Issue 5, 17 May 2022, Page 100634</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Lipid nanoparticle (LNP)-mRNA vaccines offer protection against COVID-19; however, multiple variant lineages caused widespread breakthrough infections. Here, we generate LNP-mRNAs specifically encoding wild-type (WT), B.1.351, and B.1.617 SARS-CoV-2 spikes, and systematically study their immune responses. All three LNP-mRNAs induced potent antibody and T cell responses in animal models; however, differences in neutralization activity have been observed between variants. All three vaccines offer potent protection against in vivo challenges of authentic viruses of WA-1, Beta, and Delta variants. Single-cell transcriptomics of WT- and variant-specific LNP-mRNA-vaccinated animals reveal a systematic landscape of immune cell populations and global gene expression. Variant-specific vaccination induces a systemic increase of reactive CD8 T cells and altered gene expression programs in B and T lymphocytes. BCR-seq and TCR-seq unveil repertoire diversity and clonal expansions in vaccinated animals. These data provide assessment of efficacy and direct systems immune profiling of variant-specific LNP-mRNA vaccination in vivo.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1021/jacs.1c05550" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">DNA-Origami NanoTrap for Studying the Selective Barriers Formed by Phenylalanine-Glycine-Rich Nucleoporins</h4>
                                                        <div class="pubauthor"><strong>Qi Shen</strong>, Taoran Tian, Qiancheng Xiong, Patrick D Ellis Fisher, Yong Xiong, Thomas J Melia, C Patrick Lusk, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Journal of the American Chemical Society, Volume 143, Issue 31, 29 July 2021, Pages 12294-12303</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>DNA nanotechnology provides a versatile and powerful tool to dissect the structure–function relationship of biomolecular machines like the nuclear pore complex (NPC), an enormous protein assembly that controls molecular traffic between the nucleus and cytoplasm. To understand how the intrinsically disordered, Phe-Gly-rich nucleoporins (FG-nups) within the NPC establish a selective barrier to macromolecules, we built a DNA-origami NanoTrap. The NanoTrap comprises precisely arranged FG-nups in an NPC-like channel, which sits on a baseplate that captures macromolecules that pass through the FG network. Using this biomimetic construct, we determined that the FG-motif type, grafting density, and spatial arrangement are critical determinants of an effective diffusion barrier. Further, we observed that diffusion barriers formed with cohesive FG interactions dominate in mixed-FG-nup scenarios. Finally, we demonstrated that the nuclear transport receptor, Ntf2, can selectively transport model cargo through NanoTraps composed of FxFG but not GLFG Nups. Our NanoTrap thus recapitulates the NPC’s fundamental biological activities, providing a valuable tool for studying nuclear transport.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s42003-021-02449-8" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">PRL3 induces polyploid giant cancer cells eliminated by PRL3-zumab to reduce tumor relapse</h4>
                                                        <div class="pubauthor"><strong>Min Thura</strong>, <strong>Zu Ye</strong>, Abdul Qader Al-Aidaroos, Qiancheng Xiong, Jun Yi Ong, Abhishek Gupta, Jie Li, Ke Guo, Koon Hwee Ang, Qi Zeng</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Communications Biology, Volume 4, Issue 1, 29 July 2021, 923</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>PRL3, a unique oncotarget, is specifically overexpressed in 80.6% of cancers. In 2003, we reported that PRL3 promotes cell migration, invasion, and metastasis. Herein, firstly, we show that PRL3 induces Polyploid Giant Cancer Cells (PGCCs) formation. PGCCs constitute stem cell-like pools to facilitate cell survival, chemo-resistance, and tumor relapse. The correlations between PRL3 overexpression and PGCCs attributes raised possibilities that PRL3 could be involved in PGCCs formation. Secondly, we show that PRL3+ PGCCs co-express the embryonic stem cell markers SOX2 and OCT4 and arise mainly due to incomplete cytokinesis despite extensive DNA damage. Thirdly, we reveal that PRL3+ PGCCs tolerate prolonged chemotherapy-induced genotoxic stress via suppression of the pro-apoptotic ATM DNA damage-signaling pathway. Fourthly, we demonstrated PRL3-zumab, a First-in-Class humanized antibody drug against PRL3 oncotarget, could reduce tumor relapse in ‘tumor removal’ animal model. Finally, we confirmed that PGCCs were enriched in relapse tumors versus primary tumors. PRL3-zumab has been approved for Phase 2 clinical trials in Singapore, US, and China to block all solid tumors. This study further showed PRL3-zumab could potentially serve an ‘Adjuvant Immunotherapy’ after tumor removal surgery to eliminate PRL3+ PGCC stem-like cells, preventing metastasis and relapse.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41557-021-00667-5" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Sorting sub-150-nm liposomes of distinct sizes by DNA-brick-assisted centrifugation</h4>
                                                        <div class="pubauthor"><strong>Yang Yang</strong>, <strong>Zhenyong Wu</strong>, Laurie Wang, Kaifeng Zhou, Kai Xia, Qiancheng Xiong, Longfei Liu, Zhao Zhang, Edwin R. Chapman, Yong Xiong, Thomas J. Melia, Erdem Karatekin, Hongzhou Gu, and Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Chemistry, Volume 13, Issue 4, 30 March 2021, Pages 335-342</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>In cells, myriad membrane-interacting proteins generate and maintain curved membrane domains with radii of curvature around or below 50 nm. To understand how such highly curved membranes modulate specific protein functions, and vice versa, it is imperative to use small liposomes with precisely defined attributes as model membranes. Here, we report a versatile and scalable sorting technique that uses cholesterol-modified DNA ‘nanobricks’ to differentiate hetero-sized liposomes by their buoyant densities. This method separates milligrams of liposomes, regardless of their origins and chemical compositions, into six to eight homogeneous populations with mean diameters of 30–130 nm. We show that these uniform, leak-resistant liposomes serve as ideal substrates to study, with an unprecedented resolution, how membrane curvature influences peripheral (ATG3) and integral (SNARE) membrane protein activities. Compared with conventional methods, our sorting technique represents a streamlined process to achieve superior liposome size uniformity, which benefits research in membrane biology and the development of liposomal drug-delivery systems.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1002/anie.201915555" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">DNA Origami Post‐Processing by CRISPR‐Cas12a</h4>
                                                        <div class="pubauthor"><strong>Qiancheng Xiong</strong>, <strong>Chun Xie</strong>, Zhao Zhang, Longfei Liu, John T Powell, Qi Shen, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Angewandte Chemie International Edition in English, Volume 59, Issue 10, 02 March 2020, Pages 3956-3960</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Customizable nanostructures built through the DNA‐origami technique hold tremendous promise in nanomaterial fabrication and biotechnology. Despite the cutting‐edge tools for DNA‐origami design and preparation, it remains challenging to separate structural components of an architecture built from—thus held together by—a continuous scaffold strand, which in turn limits the modularity and function of the DNA‐origami devices. To address this challenge, here we present an enzymatic method to clean up and reconfigure DNA‐origami structures. We target single‐stranded (ss) regions of DNA‐origami structures and remove them with CRISPR‐Cas12a, a hyper‐active ssDNA endonuclease without sequence specificity. We demonstrate the utility of this facile, selective post‐processing method on DNA structures with various geometrical and mechanical properties, realizing intricate structures and structural transformations that were previously difficult to engineer. Given the biocompatibility of Cas12a‐like enzymes, this versatile tool may be programmed in the future to operate functional nanodevices in cells.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41589-019-0325-3" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">A Programmable DNA-Origami Platform for Studying Lipid Transfer between Bilayers</h4>
                                                        <div class="pubauthor"><strong>Xin Bian</strong>, <strong>Zhao Zhang</strong>, Qiancheng Xiong, Pietro De Camilli, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Chemical Biology, Volume 15, Issue 8, 18 July 2019, Pages 830–837</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Non-vesicular lipid transport between bilayers at membrane contact sites plays important physiological roles. Mechanistic insight into the action of lipid-transport proteins localized at these sites requires determination of the distance between bilayers at which this transport can occur. Here we developed DNA-origami nanostructures to organize size-defined liposomes at precise distances and used them to study lipid transfer by the synaptotagmin-like mitochondrial lipid-binding protein (SMP) domain of extended synaptotagmin 1 (E-Syt1). Pairs of DNA-ring-templated donor and acceptor liposomes were docked through DNA pillars, which determined their distance. The SMP domain was anchored to donor liposomes via an unstructured linker, and lipid transfer was assessed via a Förster resonance energy transfer (FRET)-based assay. We show that lipid transfer can occur over distances that exceed the length of an SMP dimer, which is compatible with the shuttle model of lipid transport. The DNA nanostructures developed here can also be adapted to study other processes occurring where two membranes are closely apposed to each other.</p>
                                                    </div>
                                                </div>
                                            </div>
                                        </div>
                                    </div>

                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div id="contact" class="page stellar">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                
                                <h2 class="title">Contact & Meet Me</h2>
                            
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>I would be happy to talk to you about my ideas and work. You can also contact me if you need my assistance in your research.</p>                              
                                    </div>
                                    <div class="col-md-4">
                                        <ul class="list-unstyled">
                                            <li>
                                                <strong><i class="icon-envelope"></i>&nbsp;&nbsp;</strong>
                                                <span>qiancheng.xiong (a) wyss.harvard.edu</span>
                                            </li>
                                            <li>
                                                <strong><i class="icon-twitter"></i>&nbsp;&nbsp;</strong>
                                                <span>@qcxiong</span>
                                            </li>
                                            <li>
                                                <strong><i class="icon-linkedin-sign"></i>&nbsp;&nbsp;</strong>
                                                <span><a href="https://sg.linkedin.com/in/qcxiong">qcxiong</a></span>
                                            </li>
                                                                                        <li>
                                                <strong><i class="icon-github-sign"></i>&nbsp;&nbsp;</strong>
                                                <span><a href="https://github.com/xiongqc">xiongqc</a></span>
                                            </li>
                                        </ul>    

                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section contact-lab" data-stellar-background-ratio="0.1">
                            <div class="section-container">
                                <div class="row">
                                    
                                    <div class="col-md-8">
                                        <!--<h2 class="title">At My Lab</h2>-->
                                        <!--
                                        <p>You can find me at my lab located at the Institute of Molecular and Cell Biology.</p>
                                        <p>Please email to fix an appointment.</p>
                                        -->
                                    </div>
                                    <div class="col-md-4 text-center hidden-xs hidden-sm">
                                        <i class="icon-superscript icon-huge"></i>
                                    </div>

                                </div>
                            </div>
                        </div>
                    </div>
                    
                </div>
                
                <div id="overlay"></div>
            
            </div>
        </div>
    </body>
</html>
