<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Qiancheng Xiong</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

        <meta name="description" content="Qiancheng Xiong's profile page" />
        <meta name="keywords" content="Qiancheng, Xiong, A*STAR, Cambridge, qx217, Qian Cheng, Qiancheng Xiong, Xiong Qian Cheng, Xiong Qiancheng, Qian Cheng Xiong, QC, qcxiong, QC Xiong, Q Xiong, XQC" />
        <meta name="robots" content="index, follow" />
        <meta name="googlebot" content="index, follow, max-snippet:-1, max-image-preview:large, max-video-preview:-1" />
        <meta name="bingbot" content="index, follow, max-snippet:-1, max-image-preview:large, max-video-preview:-1" />
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <meta name="msvalidate.01" content="0477ED63E0C6D968BB8B595E3C610279" />

        <script type="application/ld+json">
        {
          "@context": "https://schema.org/",
          "@type": "Person",
          "name": "Qiancheng Xiong",
          "url": "https://www.qcxiong.com/",
          "image": "https://www.qcxiong.com/img/qiancheng-xiong.jpg",
          "sameAs": [
            "https://www.linkedin.com/in/qcxiong/",
            "https://scholar.google.com/citations?user=qRvTgzgAAAAJ",
            "https://github.com/xiongqc",
            "https://orcid.org/0000-0002-9897-7391"
          ],
          "worksFor": {
            "@type": "CollegeOrUniversity",
            "name": "Yale University"
          }  
        }
        </script>

        <!--CSS styles-->
        <link rel="stylesheet" href="css/bootstrap.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">  
        <link rel="stylesheet" href="css/perfect-scrollbar-0.4.5.min.css">
        <link rel="stylesheet" href="css/magnific-popup.css">
        <link rel="stylesheet" href="css/style.css">
        <link id="theme-style" rel="stylesheet" href="css/styles/blue.css">
        <!--/CSS styles-->
	    
        <!--Javascript files-->
        <script type="text/javascript" src="js/jquery-1.10.2.js"></script>
        <script type="text/javascript" src="js/TweenMax.min.js"></script>
        <script type="text/javascript" src="js/jquery.touchSwipe.min.js"></script>
        <script type="text/javascript" src="js/jquery.carouFredSel-6.2.1-packed.js"></script>
        
        <script type="text/javascript" src="js/modernizr.custom.63321.js"></script>
        <script type="text/javascript" src="js/jquery.dropdownit.js"></script>

        <script type="text/javascript" src="js/jquery.stellar.min.js"></script>
        <script type="text/javascript" src="js/ScrollToPlugin.min.js"></script>

        <script type="text/javascript" src="js/bootstrap.min.js"></script>

        <script type="text/javascript" src="js/jquery.mixitup.min.js"></script>

        <script type="text/javascript" src="js/masonry.min.js"></script>

        <script type="text/javascript" src="js/perfect-scrollbar-0.4.5.with-mousewheel.min.js"></script>

        <script type="text/javascript" src="js/magnific-popup.js"></script>
        <script type="text/javascript" src="js/custom.js"></script>
        <!--/Javascript files-->


        
	    
	<!--Google Analytics-->
	<script>
 	 (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
 	 (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 	 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 	 })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

 	 ga('create', 'UA-92395947-1', 'auto');
 	 ga('send', 'pageview');
	</script>
	<!--/Google Analytics-->
	    
           </head>
    <body>

        <div id="wrapper">
            <a href="#sidebar" class="mobilemenu"><i class="icon-reorder"></i></a>

            <div id="sidebar">
                <div id="main-nav">
                    <div id="nav-container">
                        <div id="profile" class="clearfix">
                            <div class="portrate hidden-xs"></div>
                            <div class="title">
                                <h2>Qiancheng Xiong</h2>
                                <h3>Yale University</h3>
                            </div>
                            
                        </div>
                        <ul id="navigation">
                            <li>
                              <a href="#biography">
                                <div class="icon icon-user"></div>
                                <div class="text">About Me</div>
                              </a>
                            </li>  
                            <!--
                            <li>
                              <a href="#research">
                                <div class="icon icon-book"></div>
                                <div class="text">Research</div>
                              </a>
                            </li>
                            -->
                            <li>
                              <a href="#publications">
                                <div class="icon icon-edit"></div>
                                <div class="text">Publications</div>
                              </a>
                            </li>  
                            <li>
                              <a href="#contact">
                                <div class="icon icon-calendar"></div>
                                <div class="text">Contact</div>
                              </a>
                            </li>
                            <!--
                            <li class="external">
                              <a href="xiongqc.github.io/cv/CV_Qiancheng_Xiong caa 050118.pdf" download>
                                  <div class="icon icon-download-alt"></div>
                                  <div class="text">Curricula Vitae</div>
                              </a>
                            </li>
                            -->
                        </ul>
                    </div>        
                </div>
                
                <div class="social-icons">
                    <ul>
                        <!--
                        <li><a href="https://www.facebook.com/qiancheng"><i class="icon-facebook"></i></a></li>
                        -->
                        <li><a href="https://twitter.com/qcxiong"><i class="icon-twitter"></i></a></li>
                        <li><a href="https://www.linkedin.com/in/qcxiong"><i class="icon-linkedin"></i></a></li>
                    </ul>
                </div>    
            </div>

            <div id="main">
            
                <div id="biography" class="page home" data-pos="home">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                
                                <div class="row">
                                    <div class="col-sm-2 visible-sm"></div>
                                    <div class="col-sm-8 col-md-5">
                                        <div class="biothumb">
                                            <img alt="image" src="img/personal/bio.jpg"  class="img-responsive">
                                        </div>
                                        
                                    </div>
                                    <div class="clearfix visible-sm visible-xs"></div>
                                    
                                    <div class="col-sm-12 col-md-7">
                                        <h3 class="title">Biography</h3>
                                        <p>Qiancheng graduated from the University of Cambridge with a major in Genetics and worked as a Research Officer at the Institute of Molecular and Cell Biology (A*STAR, Singapore). He is a graduate student at Yale University and studies in the Biological and Biomedical Sciences (BBS) doctoral program. Outside the laboratory, he also experiments with coffee and cakes.</p>
                                    </div>  
                                    
                                </div>
                            </div>        
                        </div>
                    </div>

                    <div class="pagecontents">
                        <div class="section color-1">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-5 col-md-offset-1">
                                        <div class="title text-center">
                                            <h3>Work Experience</h3>
                                        </div>
                                        <ul class="ul-dates">
                                        
                                            <li>
                                                <div class="dates">
                                                    <span>2017</span>
                                                    <span>2016</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Research Officer</h4>
                                                    <p><em>Agency for Science, Technology and Research</em>, Institute of Molecular and Cell Biology</p>
                                                </div>
                                            </li>

                                        
                                            <li>
                                                <div class="dates">
                                                    <span>2012</span>
                                                    <span>2011</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Detachment Commander</h4>
                                                    <p><em>Singapore Armed Forces</em>, Singapore Armoured Regiment</p>
                                                    <a href="/testimonial/Testimonial_SAF.pdf" target="_blank">
                                                    <p>
                                                    <strong><i class="icon-thumbs-up"></i></strong>
                                                	<em>View Testimonial</em>
                                                	</p>
                                                	</a>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dates">
                                                    <span>2010</span>
                                                </div>
                                                <div class="content">
                                                    <h4>Intern</h4>
                                                    <p><em>Singapore Ministry of Defence</em>, Defence Psychology Department</p>
													<a href="/testimonial/Testimonial_MINDEF.pdf" target="_blank">
                                                    <p>
                                                    <strong><i class="icon-thumbs-up"></i></strong>
                                                	<em>View Testimonial</em>
                                                	</p>
                                                	</a>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                    <div class="col-md-5">
                                        <div class="title text-center">
                                            <h3>Education</h3>
                                        </div>
                                        <ul class="ul-card">
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">Ph.D.</span><span class="year">Expected 2023</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Doctoral Candidate in Cell Biology</p>
                                                    <p class="waht">Master of Philosophy in Cell Biology, 2020</p>
                                                    <p class="where">Yale University</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">B.A.</span><span class="year">2016</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Bachelor of Arts in Natural Sciences</p>
                                                    <p class="where">University of Cambridge</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="dy">
                                                    <span class="degree">Diploma</span><span class="year">2010</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Honours in Mathematics, Biology and Chemistry, and Major in Physics</p>
                                                    <p class="where">National University of Singapore</p> 
                                                    <p class="where">High School of Mathematics and Science</p>	
                                                </div>
                                            </li>
						                    <li>
                                                <div class="dy">
                                                    <span class="degree">Languages</span>
                                                </div>
                                                <div class="description">
                                                    <p class="waht">Chinese</p>
                                                    <p class="where">GCE O-Level Higher Chinese</p> 
                                                </div>
							
						                        <div class="description">
                                                    <p class="waht">Japanese</p>
                                                    <p class="where">GCE O-Level Japanese</p> 
                                                </div>
						                        <div class="description">
                                                    <p class="waht">French</p>
                                                    <p class="where">CEFRL A2 Level</p> 
                                                </div>
                                            </li>
                                        </ul>
                                    </div>    
                                </div>    
                            </div>
                                
                        </div>

                        <div class="section color-2">
                            <div class="section-container">
                                <div class="row">
                                    <div class="col-md-10 col-md-offset-1">
                                        <div class="title text-center">
                                            <h3>Honors, Awards and Grants</h3>
                                        </div>
                                        <ul class="timeline">
                                            <li class="open">
                                                <div class="date">2017</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">A*STAR Graduate Scholarship</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/ASTAR.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            The scholarship fully funds up to four years of PhD studies in Singapore or partially finances one abroad. Upon graduation, the scholar will also be eligible for a 2-year post-doctoral fellowship overseas.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>        
                                            <li class>
                                                <div class="date">2011</div>
                                                <div class="circle"></div>
                                                <div class="data">
                                                    <div class="subject">National Science Scholarship (BS)</div>
                                                    <div class="text row">
                                                        <div class="col-md-2">
                                                            <img alt="image" class="thumbnail img-responsive" src="img/ASTAR.jpg" >
                                                        </div>
                                                        <div class="col-md-10">
                                                            The National Science Scholarship (BS) awarded by the Agency for Science, Technology and Research (A*STAR) is an undergraduate scholarship that grooms young talents for a PhD education.
                                                        </div>
                                                    </div>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>                            
                    </div>
                </div>  
                <!--
                <div id="research" class="page">
                    <div class="pageheader">

                        <div class="headercontent">

                            <div class="section-container">
                                <h2 class="title">Research Summary</h2>
                            
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>Prof Zeng Qi identified PRL-3 in 1998 and her group later demonstrated that the phosphatase plays a causal role in various types of cancer. Together with Abdul from the group, Qiancheng is working to further elucidate the molecular details of how PRL-3 promotes cancer. </p>
                                    </div>
                                    <div class="col-md-4">
                                        <div class="subtitle text-center">
                                            <h3>Interests</h3>
                                        </div>
                                        <ul class="ul-boxed list-unstyled">
                                            <li>Genetics</li>
                                            <li>Bioinformatics</li>
                                            <li>Cancer</li>
                                            <li>Infectious Diseases</li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    
                    <div class="pagecontents">
                        <div class="section color-2">
                            <div class="section-container">
                                <div class="title text-center">
                                    <h3>Research Experience</h3>
                                </div>
                                <div class="row">
                                    <div class="col-md-12">
                                        <ul class="ul-withdetails">
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p2.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Molecular Evolution of Rotaviruses</h3>
                                                            <p><em>January - March 2016</em></p>
                                                            <p>Department of Veterinary Medicine, University of Cambridge</p>
                                                            <p>Lab of Simon Frost</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Rotavirus is a major cause of acute gastroenteritis and child mortality worldwide. Through estimating selection pressures and mutation rates at both the gene level and the site-by-site level, I demonstrated evolutionary differences between rotavirus species, proteins and hosts. In particular, higher mutation rate and diversifying selection estimates for species B may facilitate its expansion to geographical regions outside Asia. Capsid protein VP6 was also found to be a possible vaccine target due to its low mutation rate and diversifying selection estimates. Separately, reconstructed effective population size and estimated mutation rates for species A showed no significant change in genetic variation after the introduction of modern vaccines in 2005, but new sites under diversifying selection were identified on its gene for the vaccine targeted protein VP4. Finally, reconstructed phylogenies showed ongoing zoonotic transmission in species A but not in species B and C.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p3.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Transcriptional Regulation by RNA Antiterminators</h3>
                                                            <p><em>July-August 2015</em></p>
                                                            <p>Institut de Génétique et de Biologie Moléculaire et Cellulaire</p>
                                                            <p>Lab of Albert Weixlbaumer</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Transcripts of putRNAs from phage HK022 can fold into two stem-loops to inhibit RNA polymerase backtracking and facilitate antitermination. To assist the lab in understanding the molecular and structural details of this process, I cloned mutant put in vivo reporter constructs with sGFP and lacZα, and compared the effect of antitermination by wild type putRNA and their mutants using in vivo reporter assays. I also produced putRNA using in vitro transcription and purified it for the lab’s later studies.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p4.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Prevalence of Wolbachia in Moths & Phylogenetic Comparison of Wolbachia and Moths</h3>
                                                            <p><em>September-October 2014</em></p>
                                                            <p>Department of Zoology, University of Cambridge</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>Wolbachia infection provides insect hosts with virus resistance but at the cost of hosts’ fitness. To better understand the side effects of Wolbachia pest control measures, the relationship of Wolbachia with moth hosts was explored by my team using computational prevalence and phylogenetic analysis. This was performed with DNA extracted from moths caught at Madingley, as well as with sequence data from GenBank.</p>
                                                </div>
                                            </li>
                                            <li>
                                                <div class="row">
                                                    <div class="col-sm-6 col-md-3">
                                                        <div class="image">
                                                            <img alt="image" src="img/projects/p5.jpg"  class="img-responsive">
                                                            <div class="imageoverlay">
                                                                <i class="icon icon-search"></i>
                                                            </div>

                                                        </div>
                                                    </div>
                                                    <div class="col-sm-6 col-md-9">
                                                        <div class="meta">
                                                            <h3>Nuclear Translocation of NFκB in Response to Viral Protein Expression</h3>
                                                            <p><em>June-August 2014</em></p>
                                                            <p>Singapore Immunology Network, Agency for Science, Technology and Research</p>
                                                            <p>Lab of Ren Ee Chee</p>
                                                        </div>
                                                    </div>
                                                </div>
                                                <div class="details">
                                                    <p>In a co-localization study, I assisted the lab group by quantifying the relative amounts of NFκB nuclear translocation in cells that were over-expressing different variants of a viral protein using western blots. In addition, I generated plasmid constructs to assist two separate projects: to investigate how residues in a viral protein affect its function; and to create CRISPR knock-out cell lines for studying viral replication.</p>
                                                </div>
                                            </li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>    
                    </div>
                </div>
                -->
                <div id="publications" class="page">
                    <div class="page-container">
                        <div class="pagecontents">
                            <div class="headercontent">
                                <div class="section-container">
                                    <h2 class="title">Publications</h2>
                                </div>
                                <div class="section color-1" id="filters">
                                    <div class="section-container">
                                        <div class="row">
                                            
                                            <div class="col-md-3">
                                                <h3>Filter by type:</h3>
                                            </div>
                                            <div class="col-md-6">
                                                <select id="cd-dropdown" name="cd-dropdown" class="cd-select">
                                                    <option class="filter" value="all" selected>All types</option>
                                                    <option class="filter" value="jpaper">Original Research</option>
                                                    <option class="filter" value="cpaper">Commentary</option>
                                                    <!-- <option class="filter" value="bookchapter">Book Chapter</option> -->
                                                    <!-- <option class="filter" value="book">Book</option> -->
                                                    <option class="filter"value="report">Review</option>
                                                    <option class="filter" value="tpaper">Patent</option>
                                                </select>
                                            </div>
                                            
                                            <div class="col-md-3" id="sort">
                                                <span>Sort by year:</span>
                                                <div class="btn-group pull-right"> 

                                                    <button type="button" data-sort="data-year" data-order="desc" class="sort btn btn-default"><i class="icon-sort-by-order"></i></button>
                                                    <button type="button" data-sort="data-year" data-order="asc" class="sort btn btn-default"><i class="icon-sort-by-order-alt"></i></button>
                                                </div>
                                            </div>    
                                        </div>
                                    </div>
                                </div>
                            </div>
                            <div class="section color-2" id="pub-grid">
                                <div class="section-container">
                                    <div class="row">
                                        <div class="col-md-12">
                                            <div class="pitems">
                                            <!-- Add new items after this line -->
                                                <div class="item mix report" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41467-022-30878-4" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Omicron-specific mRNA vaccination alone and as a heterologous booster against SARS-CoV-2</h4>
                                                        <div class="pubauthor"><strong>Zhenhao Fang</strong>, <strong>Lei Peng</strong>, Renata Filler, Kazushi Suzuki, Andrew McNamara, Qianqian Lin, Paul A. Renauer, Luojia Yang, Bridget Menasche, Angie Sanchez, Ping Ren, Qiancheng Xiong, Madison Strine, Paul Clark, Chenxiang Lin, Albert I. Ko, Nathan D. Grubaugh, Craig B. Wilen, Sidi Chen</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Communications, Volume 13, Issue 1, 06 June 2022, Page 3250</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>The Omicron variant of SARS-CoV-2 recently swept the globe and showed high level of immune evasion. Here, we generate an Omicron-specific lipid nanoparticle (LNP) mRNA vaccine candidate, and test its activity in animals, both alone and as a heterologous booster to WT mRNA vaccine. Our Omicron-specific LNP-mRNA vaccine elicits strong antibody response in vaccination-naïve mice. Mice that received two-dose WT LNP-mRNA show a > 40-fold reduction in neutralization potency against Omicron than WT two weeks post boost, which further reduce to background level after 3 months. The WT or Omicron LNP-mRNA booster increases the waning antibody response of WT LNP-mRNA vaccinated mice against Omicron by 40 fold at two weeks post injection. Interestingly, the heterologous Omicron booster elicits neutralizing titers 10-20 fold higher than the homologous WT booster against Omicron variant, with comparable titers against Delta variant. All three types of vaccination, including Omicron alone, WT booster and Omicron booster, elicit broad binding antibody responses against SARS-CoV-2 WA-1, Beta, Delta variants and SARS-CoV. These data provide direct assessments of an Omicron-specific mRNA vaccination in vivo, both alone and as a heterologous booster to WT mRNA vaccine.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix report" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1021/acs.accounts.2c00234" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Frame-Guided Assembly of Amphiphiles</h4>
                                                        <div class="pubauthor"><strong>Yuanchen Dong</strong>, Yang Yang, Chenxiang Lin, Dongsheng Liu</div>
                                                        <div class="pubcite"><span class="label label-success">Review</span>Accounts of Chemical Research, Volume 55, Issue 14, 05 July 2022, Page 1938–1948</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Amphiphiles tend to self-assemble into various structures and morphologies in aqueous environments (e.g., micelles, tubes, fibers, vesicles, and lamellae). These assemblies and their properties have made significant impact in traditional chemical industries, e.g., increasing solubility, decreasing surface tension, facilitating foaming, etc. It is well-known that the molecular structure and its environment play a critical role in the assembly process, and many theories, including critical packing factor, thermodynamic models, etc., have been proposed to explain and predict the assembly morphology. It has been recognized that the morphology of the amphiphilic assembly plays important roles in determining the functions, such as curvature-dependent biophysical (e.g., liposome fusion and fission) and biochemical (e.g., lipid metabolism and membrane protein trafficking) processes, size-related EPR (enhanced permeability and retention) effects, etc. Meanwhile, various nanomaterials have promised great potential in directing the arrangement of molecules, thus generating unique functions. Therefore, control over the amphiphilic morphology is of great interest to scientists, especially in nanoscale with the assistance of functional nanomaterials. However, how to precisely manipulate the sizes and shapes of the assemblies is challenged by the entropic nature of the hydrophobic interaction. Inspired by the “cytoskeleton–membrane protein–lipid bilayer” principle of the cell membrane, a strategy termed “frame-guided assembly (FGA)” has been proposed and developed to direct the arrangement of amphiphiles. The FGA strategy welcomes various nanomaterials with precisely controlled properties to serve as scaffolds. By introducing scattered hydrophobic molecules, which are defined as either leading hydrophobic groups (LHGs) or nucleation seeds onto a selected scaffold, a discontinuous hydrophobic trace along the scaffold can be outlined, which will further guide the amphiphiles in the system to grow and form customized two- or three-dimensional (2D/3D) membrane geometries. Topologically, the supporting frame can be classified as three types including inner-frame, outer-frame, and planar-frame. Each type of FGA assembly possesses particular advantages: (1) The inner-frame, similar to endoskeletons of many cellular structures, steadily supports the membrane from the inside and exposes the full surface area outside. (2) The outer-frame, on the other hand, molds and constrains the membrane-wrapped vesicles to regulate their size and shape. It also allows postengineering of the frame to precisely decorate and dynamically manipulate the membrane. (3) The planar-frame mediates the growth of the 2D membrane that profits from the scanning-probe microscopic characterization and benefits the investigation of membrane proteins. In this Account, we introduce the recent progress of frame-guided assembly strategy in the preparation of customized amphiphile assemblies, evaluate their achievements and limitations, and discuss prospective developments and applications. The basic principle of FGA is discussed, and the morphology controllability is summarized in the inner-, outer-, and planar-frame categories. As a versatile strategy, FGA is able to guide different types of amphiphiles by designing specific LHGs for given molecular structures. The mechanism of FGA is then discussed systematically, including the driving force of the assembly, density and distribution of the LHGs, amphiphile concentration, and the kinetic process. Furthermore, the applications of FGA have been developed for liposome engineering, membrane protein incorporation, and drug delivery, which suggest the huge potential of FGA in fabricating novel and functional complexes.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1016/j.xcrm.2022.100634" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Variant-specific vaccination induces systems immune responses and potent <em>in vivo</em> protection against SARS-CoV-2</h4>
                                                        <div class="pubauthor"><strong>Lei Peng</strong>, <strong>Paul A Renauer</strong>, <strong>Arya Ökten</strong>, <strong>Zhenhao Fang</strong>, <strong>Jonathan J Park</strong>, <strong>Xiaoyu Zhou</strong>, Qianqian Lin, Matthew B Dong, Renata Filler, Qiancheng Xiong, Paul Clark, Chenxiang Lin, Craig B Wilen, Sidi Chen</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Cell Reports Medicine, Volume 3, Issue 5, 17 May 2022, Page 100634</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Lipid nanoparticle (LNP)-mRNA vaccines offer protection against COVID-19; however, multiple variant lineages caused widespread breakthrough infections. Here, we generate LNP-mRNAs specifically encoding wild-type (WT), B.1.351, and B.1.617 SARS-CoV-2 spikes, and systematically study their immune responses. All three LNP-mRNAs induced potent antibody and T cell responses in animal models; however, differences in neutralization activity have been observed between variants. All three vaccines offer potent protection against in vivo challenges of authentic viruses of WA-1, Beta, and Delta variants. Single-cell transcriptomics of WT- and variant-specific LNP-mRNA-vaccinated animals reveal a systematic landscape of immune cell populations and global gene expression. Variant-specific vaccination induces a systemic increase of reactive CD8 T cells and altered gene expression programs in B and T lymphocytes. BCR-seq and TCR-seq unveil repertoire diversity and clonal expansions in vaccinated animals. These data provide assessment of efficacy and direct systems immune profiling of variant-specific LNP-mRNA vaccination in vivo.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2022">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41592-022-01464-9" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Fluorogenic DNA-PAINT for faster, low-background super-resolution imaging</h4>
                                                        <div class="pubauthor"><strong>Kenny K. H. Chung</strong>, <strong>Zhao Zhang</strong>, <strong>Phylicia Kidd</strong>, Yongdeng Zhang, Nathan D. Williams, Bennett Rollins, Yang Yang, Chenxiang Lin, David Baddeley, Joerg Bewersdorf</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Nature Methods, Volume 19, 02 May 2022, Pages 554-559</div>
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>DNA-based points accumulation for imaging in nanoscale topography (DNA-PAINT) is a powerful super-resolution microscopy method that can acquire high-fidelity images at nanometer resolution. It suffers, however, from high background and slow imaging speed, both of which can be attributed to the presence of unbound fluorophores in solution. Here we present two-color fluorogenic DNA-PAINT, which uses improved imager probe and docking strand designs to solve these problems. These self-quenching single-stranded DNA probes are conjugated with a fluorophore and quencher at the terminals, which permits an increase in fluorescence by up to 57-fold upon binding and unquenching. In addition, the engineering of base pair mismatches between the fluorogenic imager probes and docking strands allowed us to achieve both high fluorogenicity and the fast binding kinetics required for fast imaging. We demonstrate a 26-fold increase in imaging speed over regular DNA-PAINT and show that our new implementation enables three-dimensional super-resolution DNA-PAINT imaging without optical sectioning.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix tpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://patents.google.com/patent/US20210267894A1/en" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">DNA brick-assisted liposome sorting</h4>
                                                        <div class="pubauthor">Chenxiang Lin, Yang Yang</div>
                                                        <div class="pubcite"><span class="label label-success">Patent</span>US Patent App. 17/162,008, 02 September 2021</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>A method for producing uniform-size liposomes is provided. The liposomes are coated with a sorting agent to yield a plurality of density-modified liposomes of different sizes. These liposomes are then separated using a densitometric method. The sorting agent includes both a density-modifying moiety and a targeting moiety.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1021/jacs.1c05550" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">DNA-Origami NanoTrap for Studying the Selective Barriers Formed by Phenylalanine-Glycine-Rich Nucleoporins</h4>
                                                        <div class="pubauthor"><strong>Qi Shen</strong>, Taoran Tian, Qiancheng Xiong, Patrick D Ellis Fisher, Yong Xiong, Thomas J Melia, C Patrick Lusk, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>Journal of the American Chemical Society, Volume 143, Issue 31, 29 July 2021, Pages 12294-12303</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>DNA nanotechnology provides a versatile and powerful tool to dissect the structure–function relationship of biomolecular machines like the nuclear pore complex (NPC), an enormous protein assembly that controls molecular traffic between the nucleus and cytoplasm. To understand how the intrinsically disordered, Phe-Gly-rich nucleoporins (FG-nups) within the NPC establish a selective barrier to macromolecules, we built a DNA-origami NanoTrap. The NanoTrap comprises precisely arranged FG-nups in an NPC-like channel, which sits on a baseplate that captures macromolecules that pass through the FG network. Using this biomimetic construct, we determined that the FG-motif type, grafting density, and spatial arrangement are critical determinants of an effective diffusion barrier. Further, we observed that diffusion barriers formed with cohesive FG interactions dominate in mixed-FG-nup scenarios. Finally, we demonstrated that the nuclear transport receptor, Ntf2, can selectively transport model cargo through NanoTraps composed of FxFG but not GLFG Nups. Our NanoTrap thus recapitulates the NPC’s fundamental biological activities, providing a valuable tool for studying nuclear transport.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1371/journal.pbio.3001314" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">FisB relies on homo-oligomerization and lipid binding to catalyze membrane fission in bacteria</h4>
                                                        <div class="pubauthor"><strong>Ane Landajuela</strong>, <strong>Martha Braun</strong>, Christopher D. A. Rodrigues, Alejandro Martínez-Calvo, Thierry Doan, Florian Horenkamp, Anna Andronicos, Vladimir Shteyn, Nathan D. Williams, Chenxiang Lin, Ned S. Wingreen, David Z. Rudner, Erdem Karatekin</div>
                                                        <div class="pubcite"><span class="label label-success">Original Research</span>PLoS Biology, Volume 19, Issue 6, 29 June 2021, e3001314</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Little is known about mechanisms of membrane fission in bacteria despite their requirement for cytokinesis. The only known dedicated membrane fission machinery in bacteria, fission protein B (FisB), is expressed during sporulation in Bacillus subtilis and is required to release the developing spore into the mother cell cytoplasm. Here, we characterized the requirements for FisB-mediated membrane fission. FisB forms mobile clusters of approximately 12 molecules that give way to an immobile cluster at the engulfment pole containing approximately 40 proteins at the time of membrane fission. Analysis of FisB mutants revealed that binding to acidic lipids and homo-oligomerization are both critical for targeting FisB to the engulfment pole and membrane fission. Experiments using artificial membranes and filamentous cells suggest that FisB does not have an intrinsic ability to sense or induce membrane curvature but can bridge membranes. Finally, modeling suggests that homo-oligomerization and trans-interactions with membranes are sufficient to explain FisB accumulation at the membrane neck that connects the engulfment membrane to the rest of the mother cell membrane during late stages of engulfment. Together, our results show that FisB is a robust and unusual membrane fission protein that relies on homo-oligomerization, lipid binding, and the unique membrane topology generated during engulfment for localization and membrane scission, but surprisingly, not on lipid microdomains, negative-curvature lipids, or curvature sensing.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2021">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41557-021-00667-5" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">Sorting sub-150-nm liposomes of distinct sizes by DNA-brick-assisted centrifugation</h4>
                                                        <div class="pubauthor"><strong>Yang Yang</strong>, <strong>Zhenyong Wu</strong>, Laurie Wang, Kaifeng Zhou, Kai Xia, Qiancheng Xiong, Longfei Liu, Zhao Zhang, Edwin R. Chapman, Yong Xiong, Thomas J. Melia, Erdem Karatekin, Hongzhou Gu, and Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Journal Paper</span>Nature Chemistry, Volume 13, Issue 4, 30 March 2021, Pages 335-342</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>In cells, myriad membrane-interacting proteins generate and maintain curved membrane domains with radii of curvature around or below 50 nm. To understand how such highly curved membranes modulate specific protein functions, and vice versa, it is imperative to use small liposomes with precisely defined attributes as model membranes. Here, we report a versatile and scalable sorting technique that uses cholesterol-modified DNA ‘nanobricks’ to differentiate hetero-sized liposomes by their buoyant densities. This method separates milligrams of liposomes, regardless of their origins and chemical compositions, into six to eight homogeneous populations with mean diameters of 30–130 nm. We show that these uniform, leak-resistant liposomes serve as ideal substrates to study, with an unprecedented resolution, how membrane curvature influences peripheral (ATG3) and integral (SNARE) membrane protein activities. Compared with conventional methods, our sorting technique represents a streamlined process to achieve superior liposome size uniformity, which benefits research in membrane biology and the development of liposomal drug-delivery systems.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2020">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1002/anie.201915555" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">DNA Origami Post‐Processing by CRISPR‐Cas12a</h4>
                                                        <div class="pubauthor"><strong>Qiancheng Xiong</strong>, <strong>Chun Xie</strong>, Zhao Zhang, Longfei Liu, John T Powell, Qi Shen, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Journal Paper</span>Angewandte Chemie International Edition in English, Volume 59, Issue 10, 02 March 2020, Pages 3956-3960</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Customizable nanostructures built through the DNA‐origami technique hold tremendous promise in nanomaterial fabrication and biotechnology. Despite the cutting‐edge tools for DNA‐origami design and preparation, it remains challenging to separate structural components of an architecture built from—thus held together by—a continuous scaffold strand, which in turn limits the modularity and function of the DNA‐origami devices. To address this challenge, here we present an enzymatic method to clean up and reconfigure DNA‐origami structures. We target single‐stranded (ss) regions of DNA‐origami structures and remove them with CRISPR‐Cas12a, a hyper‐active ssDNA endonuclease without sequence specificity. We demonstrate the utility of this facile, selective post‐processing method on DNA structures with various geometrical and mechanical properties, realizing intricate structures and structural transformations that were previously difficult to engineer. Given the biocompatibility of Cas12a‐like enzymes, this versatile tool may be programmed in the future to operate functional nanodevices in cells.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2019">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="icon-expand-alt"></i>
                                                            </a>
                                                            <a href="https://doi.org/10.1038/s41589-019-0325-3" class="tooltips" title="External link" target="_blank">
                                                                <i class="icon-external-link"></i>
                                                            </a>
                                                            <!--
                                                            <a href="#.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="icon-cloud-download"></i>
                                                            </a>
                                                            -->
                                                        </div>

                                                        <h4 class="pubtitle">A Programmable DNA-Origami Platform for Studying Lipid Transfer between Bilayers</h4>
                                                        <div class="pubauthor"><strong>Xin Bian</strong>, <strong>Zhao Zhang</strong>, Qiancheng Xiong, Pietro De Camilli, Chenxiang Lin</div>
                                                        <div class="pubcite"><span class="label label-success">Journal Paper</span>Nature Chemical Biology, Volume 15, Issue 8, 18 July 2019, Pages 830–837</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Non-vesicular lipid transport between bilayers at membrane contact sites plays important physiological roles. Mechanistic insight into the action of lipid-transport proteins localized at these sites requires determination of the distance between bilayers at which this transport can occur. Here we developed DNA-origami nanostructures to organize size-defined liposomes at precise distances and used them to study lipid transfer by the synaptotagmin-like mitochondrial lipid-binding protein (SMP) domain of extended synaptotagmin 1 (E-Syt1). Pairs of DNA-ring-templated donor and acceptor liposomes were docked through DNA pillars, which determined their distance. The SMP domain was anchored to donor liposomes via an unstructured linker, and lipid transfer was assessed via a Förster resonance energy transfer (FRET)-based assay. We show that lipid transfer can occur over distances that exceed the length of an SMP dimer, which is compatible with the shuttle model of lipid transport. The DNA nanostructures developed here can also be adapted to study other processes occurring where two membranes are closely apposed to each other.</p>
                                                    </div>
                                                </div>
                                            </div>
                                        </div>
                                    </div>

                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div id="contact" class="page stellar">
                    <div class="pageheader">
                        <div class="headercontent">
                            <div class="section-container">
                                
                                <h2 class="title">Contact & Meet Me</h2>
                            
                                <div class="row">
                                    <div class="col-md-8">
                                        <p>I would be happy to talk to you about my ideas and work. You can also contact me if you need my assistance in your research.</p>                              
                                    </div>
                                    <div class="col-md-4">
                                        <ul class="list-unstyled">
                                            <li>
                                                <strong><i class="icon-envelope"></i>&nbsp;&nbsp;</strong>
                                                <span>qiancheng.xiong@yale.edu</span>
                                            </li>
                                            <li>
                                                <strong><i class="icon-skype"></i>&nbsp;&nbsp;</strong>
                                                <span>qiancheng.xiong</span>
                                            </li>
                                            <li>
                                                <strong><i class="icon-twitter"></i>&nbsp;&nbsp;</strong>
                                                <span>@qcxiong</span>
                                            </li>
                                            <li>
                                                <strong><i class="icon-linkedin-sign"></i>&nbsp;&nbsp;</strong>
                                                <span><a href="https://sg.linkedin.com/in/qcxiong">qcxiong</a></span>
                                            </li>
                                                                                        <li>
                                                <strong><i class="icon-github-sign"></i>&nbsp;&nbsp;</strong>
                                                <span><a href="https://github.com/xiongqc">xiongqc</a></span>
                                            </li>
                                        </ul>    

                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="pagecontents">
                        <div class="section contact-lab" data-stellar-background-ratio="0.1">
                            <div class="section-container">
                                <div class="row">
                                    
                                    <div class="col-md-8">
                                        <h2 class="title">At My Lab</h2>
                                        <!--
                                        <p>You can find me at my lab located at the Institute of Molecular and Cell Biology.</p>
                                        -->
                                        <p>I am at work every day from 9:00am until 6:00 pm, but you may consider an email to fix an appointment.</p>
                                    </div>
                                    <div class="col-md-4 text-center hidden-xs hidden-sm">
                                        <i class="icon-superscript icon-huge"></i>
                                    </div>

                                </div>
                            </div>
                        </div>
                    </div>
                    
                </div>
                
                <div id="overlay"></div>
            
            </div>
        </div>
    </body>
</html>
